src/payment/payment-method-ids.js

```diff
@@ -5,5 +5,6 @@ export const BRAINTREE_VISACHECKOUT = 'braintreevisacheckout';
 export const BRAINTREE_GOOGLEPAY = 'googlepaybraintree';
 
 export const PAYPAL_COMMERCE = 'paypalcommerce';
+export const PAYPAL_COMMERCE_ALTERNATIVE_METHODS = 'paypalcommercealternativemethods';
```

Comment:
We decided to put on hold this endpoint, for now, https://github.com/bigcommerce/bigcommerce/pull/37690/files#r507837862, @koa75

Reply:
@BC-ACherednichenko It's different cases. This changes need for data to bigpay during submitting payment (`api/public/v1/orders/payments`) and the same gateway - "paypalcommerce"

Reply:
ah, yes, please disregard in that case